networkit/cpp/centrality/DynBetweenness.cpp

```diff
@@ -215,59 +220,61 @@ void DynBetweenness::update(GraphEvent event) {
         std::vector<bool> enqueued(G.upperNodeIdBound(), false);
         enqueued[v] = true;
 
-        while (! stack.empty()) {
+        while (!stack.empty()) {
             node y = stack.top();
             if (!visited[y]) {
-                std::priority_queue<std::pair<double, node>, std::vector<std::pair<double,node>>, CompareDist> Qnew;
-                std::priority_queue<std::pair<double, node>, std::vector<std::pair<double,node>>, CompareDist> Qold;
+                std::priority_queue<std::pair<double, node>, std::vector<std::pair<double, node>>,
+                                    CompareDist>
+                    Qnew;
```

Comment:
That looks a bit strange. Is it possible to change this edit?

Reply:
One possibility is to have a shorter type name for the priority queue, e.g., `using PrioQ = std::priority_queue<...>;`.